app/views/case_workers/admin/allocations/_allocation.html.haml

```diff
@@ -42,14 +42,15 @@
   %h2.govuk-heading-m
     = t('.allocation_queue')
 
-  = govuk_table(class: 'app-jq-datatable', style: 'width: 100%') do
-    = govuk_table_caption(class: 'govuk-visually-hidden') do
-      = t('.table_summary')
-
-    = govuk_table_thead_collection ['',
-      t('.case_number'),
-      t('.court'),
-      t('.defendents'),
-      t('.type'),
-      t('.submitted'),
-      t('.total')]
+  = govuk_table(classes: 'app-table--responsive app-jq-datatable', html_attributes: { style: 'width: 100%' }) do |table|
```

Comment:
Is it necessary now to explicitly set `style: 'width: 100%'`? I may be wrong but I think it is included in the `govuk-table` class.

Reply:
Good point. I took the approach of changing as little about the tables as possible, but you're right that the width is set to 100% by the table class, so I've pushed a commit to remove the two instances you flagged.